server/middlewares/roleAuthorisation.js

```diff
@@ -1,38 +1,27 @@
 import 'dotenv/config';
-import jwt from 'jsonwebtoken';
-
 import Util from '../helpers/util';
 
 const util = new Util();
 
 const allow = (...roles) => {
+  
   // Define a simple role check function
   const isAllowed = (role) => roles.indexOf(role) > -1;
 
   // Define a middleware
   const authorise = (req, res, next) => {
-    // Return error if no token provided
-    if (!req.headers.authorization) {
-      util.setError(401, 'Token must be provided');
-      return util.send(res);
-    }
-
-    // Get current user role from access token
-    const { role } = jwt.verify(req.headers.authorization, process.env.JWT_KEY);
 
+    const { role } = req.userData
     // Check for current user role in allowed list
     const roleAllowed = isAllowed(role);
-
     // Return error if role not allowed
     if (!roleAllowed) {
-      util.setError(403, 'Forbidden route');
+      util.setError(403, `Forbidden route: Dear user Only ${roles} can carry this activity.`);
```

Comment:
I think it's not a good idea to reveal who are allowed the operation, if its a hacker you are giving a hint to him

Reply:
@Francois-MUGOROZI what do you think of 
`Forbidden route: Dear user you are not allowed to carry out this activity.`